### ISSUE TITLE
[Feature Request] Expansion-panels shadow: Add elevation prop

### ISSUE THREAD

**Issue by Tzahile** [vuetifyjs/vuetify#8722]
### Problem to solve
Currently (Vuetify 2) the Expansion panels component has a shadow underneath it, which sometimes is not wanted (in my case - when expansion-panels are combined in lists with flat items).

### Proposed solution
Change the shadow to be part of the `elevation` prop,
with default value of '1' or '2' (as shown in the [elevation page](https://vuetifyjs.com/en/styles/elevation)).

That way, it'll be possible to soften or harden the shadow.
___
***Similar [Feature Request](https://github.com/vuetifyjs/vuetify/issues/8204) was suggested in the past**, but not with this proposed solution, which doesn't affect the default appearance of the Expansion-panels component, yet lets the developer to modify it, if he wants.
I simulated as well elevation change (with box-shadow css modification) in the [pen](https://codepen.io/johnjleider/pen/oKwwaB) provided by John Leider and saw it can be helpful

**Comment by johnleider** [comment]
Unfortunately due to how the shadows are created we wouldn't be able to use the **elevatable** mixin here. What can be done though is having that property moved to variables so that the user can at least modify it there.

**Comment by eeriksp** [comment]
Is there any workaround to remove the shadow? Overriding some CSS manually perhaps? This worked with v1.5, but not with v2.1.5:
```css
  .v-item-group .v-expansion-panels .v-expansion-panel {
    box-shadow: none;
    -webkit-box-shadow: none;
    background-color: inherit;
  }

  .theme--light.v-expansion-panels .v-expansion-panel {
    background-color: inherit;
  }
```

**Comment by KaelWD** [comment]
@eeriksp it was in the issue body https://codepen.io/johnjleider/pen/oKwwaB

**Comment by KaelWD** [comment]
Resolved by #13482
